telemetry/telegraf/convert_test.go

```diff
@@ -186,14 +214,6 @@ func TestConverter_Convert_NumFrameFields_LabelsColumn(t *testing.T) {
 	require.JSONEqf(t, string(frameJSON), string(want), "not matched with golden file")
 }
 
-func TestConverter_Convert_LabelsColumn_MixedNumberTypes_Panics(t *testing.T) {
```

Comment:
this tested explicitly that chaning the type panics -- silently replacing with null is not great, but panic seems pretty bad for user enter data